Resources/Textures/Clothing/Head/Soft/bizzaresoft_flipped.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from civstation at commit https://github.com/Civ13/Civ13/commit/ec52cbb95d59b717d4d8c480b35ac133e5b58088#diff-fba188fb2db5d16e5d41985147336a8f96085f761b903e016fffd869b63e497d",
```

Comment:
why a flipped version?

Reply:
why not?

Reply:
Better ways to do it without bloating asset count, e.g. shader or spritecomponent.